.pipeline/npmw

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set +x
+type -t nvm && nvm deactivate
+export NVM_DIR="$(git rev-parse --show-toplevel)/.nvm"
+if [ ! -f "$NVM_DIR/nvm.sh" ]; then
+  mkdir -p "${NVM_DIR}"
+  curl -sSL -o- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | bash &>/dev/null
```

Comment:
Why are we installing nvm? If we're using one of the red hat images, aren't we constrained to whatever version of node they're running anyway?

Reply:
nvm is being used to run the pipeline scripts and it has nothing to do with the application.